index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Apple pie recipe</title>
+    <style>
+        img{
+            border: black solid 5px;
+            margin: 20px 0px 20px 0px;
+        }
+        strong{
+            color:rosybrown;
+        }
+    </style>
+</head>
+<body>
+    <h1><strong>Apple pie</strong></h1>
+    <img src="apple-pie.jpg" alt="Apple pie" height="300px" width="400px">
+    <h2><em>Total time</em></h2>
+    <p>Prep: 20 min. Bake: 45 min.</p>
+    <br>
+    <h2><em>Makes</em></h2>
+    <p>8 servings</p>
+    <br>
+    <h2><em>Ingredients</em></h2>
+    <ul>
+        <li>1/2 cup <strong>sugar</strong></li>
+        <li>1/2 cup packed <strong>brown sugar</strong></li>
+        <li>3 tablespoons all-purpose <strong>flour</strong></li>
+        <li>1 teaspoon ground <strong>cinnamon</strong></li>
+        <li>1/4 teaspoon ground <strong>ginger</strong></li>
+        <li>1/4 teaspoon ground <strong>nutmeg</strong></li>
+        <li>6 to 7 cups thinly sliced peeled tart <strong>apples</strong></li>
+        <li>1 tablespoon <strong>lemon juice</strong></li>
+        <li><strong>Pastry</strong> for double-crust pie (9 inches)</li>
+        <li>1 tablespoon <strong>butter</strong></li>
+        <li>1 large <strong>egg white</strong></li>
+        <li>Additional sugar</li>
+    </ul>
+    <br>
+    <h2><em>Directions</em></h2>
+    <p>In a small bowl, combine the sugars, flour and spices; set aside. In a large bowl, toss apples with lemon juice. Add sugar mixture; toss to coat.</p>
+    <p>Beat egg white until foamy; brush over crust. Sprinkle with sugar. Cover edges loosely with foil.</p>
+    <p>Bake at <em>375° for 25 minutes.</em> Remove foil and bake until crust is golden brown and filling is bubbly, 20-25 minutes longer. Cool on a wire rack.</p>

```

Comment:
Predložio bih kod koraka izrade koristiti uređeni popis (`<ol>`) zato što je njihov redoslijed definiran i bitan za ispravan prijenos sadržaja recepta.